tools/ci_build/github/azure-pipelines/win-qnn-arm64-ci-pipeline.yml

```diff
@@ -49,30 +46,16 @@ jobs:
       versionSpec: 6.4.x
      
   - task: PythonScript@0
-    displayName: 'Generate cmake config'
+    displayName: 'Build'
```

Comment:
You can put the three tasks together in a single powershell script. Then it would be easier to reproduce pipeline errors locally. 

Most other pipelines extract the "VSBuild@1" part out because:
1. make build errors easier to find since logs of the VSBUILD task are integrated well into Azure DevOps web UI.
2. Our Prefast task needs to get log event from a VSBUILD task.

Reply:
cool. thanks for the tips. i suggest we get this change in asap, and refactor it subsequently.
moving to arm native builds saves ~30 minutes build time vs x64 emulation (cross-compiled to arm64).